resources/migrations/0070_resource_reservable_delay_days.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
```

Comment:
It would be better to place all the migration operations (2x RenameField, 2x AddField) in a single migration.

Reply:
Migrations merged to one file in 95a4736.